source/_integrations/meteo_france.markdown

```diff
@@ -55,7 +55,7 @@ All the following sensors will be created :
 |`uv`|The current UV index|Yes|
 |`weather_alert`|Weather alert status ([see note below](#about-weather_alert-sensor))|Yes|
 |`wind_gust`|The current wind gust speed in km/h|No|
-|`wind_speed`|The current wind speed in km/h|No|
+|`wind_speed`|The current wind speed in m/s|No|
```

Comment:
I don't think this is correct, the core PR only changed the native unit of the weather platform.

Reply:
See my comment above. With unit auto-conversion in weather entities IMO it doesn't make sense to specify units at all in docs.

Reply:
So what should I do? I’ve changed it because I removed the conversion to km/h that was done in the code. So now the wind speed by default is in m/s

Reply:
> I don't think this is correct, the core PR only changed the native unit of the weather platform, this is about the unit of automatically created sensors.

I’ve also removed the conversion to km/h from m/s in the code when migratin to native_*

Reply:
What you removed was related to the meteo_france weather entity, but the documentation is related to the sensors which don't automatically convert units and have not changed.
As I understand it the documentation does not need to be modified, and this PR can simply be closed without merging.

Reply:
You are right, my mistake